data-serving/data-service/api/openapi.yaml

```diff
@@ -0,0 +1,56 @@
+openapi: 3.0.3
+info:
+  title: EPID data service
+  description: Server for CRUD operations on EPID line-list data
+  termsOfService: "https://www.healthmap.org/about/termsofuse/"
+  contact:
+    email: info@healthmap.org
+  license:
+    name: MIT
+  version: 1.0.0
+paths:
+  "/api/cases/{id}":
+    summary: Line-list cases
+    get:
+      summary: Gets a specific case
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+          description: The case ID
+      responses:
+        "200":
```

Comment:
Another nit, but we're also returning 201 for createCase, IIRC.

Reply:
I think this list of response codes is just for the GET request (slight drawback of YAML is hard to tell indentations) -- when i add the POST, i'll be sure to use 201

Reply:
Right, sorry. Note to self: don't comment just based on PR code snippets. ;-)